### PR TITLE
Print replicated and starting bundle counds. Print hints.

### LINKED ISSUE #28: Bundle Info Run vs Start
Bundle info should provide a `#Start` column in addition to `#Run` that distinguishes the number of bundles starting from the number that are running. At present `#Run` reports a bundle as running even when it is starting.

### PR DESCRIPTION
**Pull request by 2m**
Fixes #28 and #26.

### CODE REVIEW

File: src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala [hunk 1]
```diff
@@ -105,7 +105,7 @@ object Column {
       bundles.map { bundle =>
         bundle.bundleInstallations.map { node =>
           val nodeIpAndPort = node.uniqueAddress.address.toString.dropWhile(_ != '@').drop(1)
-          if (bundle.bundleExecutions.map(_.host) contains node.uniqueAddress.address.host.get)
+          if (bundle.bundleExecutions.filter(_.isStarted).map(_.host) contains node.uniqueAddress.address.host.get)
```
Comment (2m) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Yes, filter + map = collect, but this is more elegant IMO than:
```
import ....BundleExecution
...
... collect { case BundleExecution(host, isStarted) if isStarted => host }
```

Comment (hseeberger) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: But your solution iterates the collection twice!
Also, you can improve the function given to collect:
``` scala
collect { case BundleExecution(host, true) => host }
```

Comment (hseeberger) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Alternatively you can use `withFilter` which avoids the double iteration.

Comment (2m) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Yes. But only the values left out after the filter. :) Okay, I can change to collect.

Comment (2m) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Ahh. Yes. With filter is nice here.

File: src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala [hunk 2]
```diff
@@ -105,7 +105,7 @@ object Column {
       bundles.map { bundle =>
         bundle.bundleInstallations.map { node =>
           val nodeIpAndPort = node.uniqueAddress.address.toString.dropWhile(_ != '@').drop(1)
-          if (bundle.bundleExecutions.map(_.host) contains node.uniqueAddress.address.host.get)
+          if (bundle.bundleExecutions.withFilter(_.isStarted).map(_.host) contains node.uniqueAddress.address.host.get)
```
Comment (huntc) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Oh, how did @hseeberger let you get away with the infix notation? ;-)

Comment (hseeberger) [inline src/main/scala/com/typesafe/conductr/sbt/console/Columns.scala]: Ouch, what a terrible mistake ;-)

**Pull request merged**